src/sasa.cpp

```diff
@@ -233,4 +239,208 @@ TEST_CASE("[Faunus] SASAPBC") {
     }
 }
 
+template <typename CellList_T>
+SASACellList<CellList_T>::SASACellList(Space& spc, double probe_radius, int slices_per_atom)
+    : SASABase(spc, probe_radius, slices_per_atom) {}
+
+template <typename CellList_T>
+SASACellList<CellList_T>::SASACellList(const json& j, Space& spc)
+    : SASABase(spc, j.value("radius", 1.4) * 1.0_angstrom, j.value("slices", 20)) {}
+
+template <typename CellList_T> void SASACellList<CellList_T>::init(Space& spc) {
+
+    cell_offsets.clear();
+    for (auto i = -1; i <= 1; ++i) {
+        for (auto j = -1; j <= 1; ++j) {
+            for (auto k = -1; k <= 1; ++k) {
+                cell_offsets.push_back(CellCoord(i, j, k));
+            }
+        }
+    }
+
+    radii.clear();
```

Comment:
I wanted to use that as you suggest but without the `auto` because radii is already declared as a member, but then the compiler complains:

error: no match for ‘operator=’ (operand types are ‘std::vector<double>’ and ‘ranges::transform_view<ranges::ref_view<std::vector<Faunus::Particle> >, Faunus::SASA::SASA::init(Faunus::Space&)::<lambda(auto:46&)> >’)
  146 |     radii = spc.particles | ranges::cpp20::views::transform([](auto &i){return 0.5 * i.traits().sigma;});